config/nimdoc.cfg

```diff
@@ -450,7 +595,7 @@ h1 {
   font-size: 1.8em;
   font-weight: 400;
   padding-bottom: .25em;
-  border-bottom: 1px solid #aaa;
+  border-bottom: 6px solid var(--third-background);
```

Comment:
Note that support for these isn't 100% https://caniuse.com/#feat=css-variables

Reply:
Some of the listed browser (versions) are either mobile or are really really outdated. I doubt that you'd go ahead and actually use Nim's documentation on mobile. And even if you would do that, **most** browsers for _mobile_ now support that.

Reply:
PC:
- Edge (Since Oct 17, 2017)
- Firefox (Since Jul 22, 2014)
- Chrome (Since Mar 3, 2016)
- Safari (Since Mar 21, 2016)
- Opera (Since Mar 7, 2016)

Mobile:
- Safari & Chrome for iOS (Since Mar 21, 2016)
- Android Browser (Since Aug 26, 2019 - Android Browser has huge gaps between releases, see the page you linked, click on "Date Relative")
- Opera Mobile (Since Sep 23, 2016)
- Chrome for Android (Since Oct 22, 2019)
- Firefox for Android (Since Sep 3, 2019)
- UC Browser for Android (Since Aug 17, 2016)
- Samsumg Internet (Since Dec 16, 2016)
- KaiOS Browser (Since Jun 1, 2018)

Not supported by:
- Opera Mini (Compensated by Opera Mobile)
- QQ Browser (Latest release Jan 1, 2017 - No releases ever since according to _caniuse.com_, data might be outdated)
- Baidu Browser (Latest release Apr 1, 2017 - No releases ever since according to _caniuse.com_, data might be outdated)